### DISCUSSION TITLE
Show your main menu here! (post screenshot)

### DISCUSSION THREAD

**Discussion by BazzaCuda** [BazzaCuda/CustomMenu#25] [Show and tell]
Post a screenshot of your main menu to show others how you've used `CustomMenu`!

**Comment by Laurens1964** [comment]
Hi Baz,

Here is mine. Its is compact but for me very practical.
I copied all kind of DLL files with icons in it to the CustomMenu directory. That's saves me al lot of clicking.

"Browse c:\" is the directory browser of my c-drive.
Further on in the menu you see the section "C-Root", "Downloads", "Admin", etc. That is to jump directly to those (for me important) directories.
I also use the direct jump to settings a lot.
If you have questions, leave a comment.

![Shot](https://github.com/user-attachments/assets/df48313e-1bf4-4f74-80dd-c2abfe2169ee)
![Dir](https://github.com/user-attachments/assets/0181e9e1-9572-4bbe-8d38-76b1c6f9788a)

**Reply by BazzaCuda** [reply]
Nice. I like the idea of copying the icon DLLs. Very handy.

**Comment by Laurens1964** [comment]
I have an update of my main CustomMenu.
![2024-08-09_140727](https://github.com/user-attachments/assets/208a332d-ad6b-4860-84c4-b022aa5a69cb)

And to be honest, copying the icon DLL's save a lot of searching and clicking. I would recommend it!
Another tip is to make a copy of the CustomMenu.ini to CustomMenu.bak each time you make some modifications to the menu. Better safe than sorry!

**Reply by BazzaCuda** [reply]
Looks good. It might be useful to have the option of copying the standard icon DLLs to the CustomMenu folder. I'll have a think about that.

**Comment by Laurens1964** [comment]
Here a new screenshot with release 2.01. The use of colors is great and the automatic backup of the ini-file is a lifesaver.

![image](https://github.com/user-attachments/assets/6ab121b5-127f-4ff5-a56f-0f70959a8b10)
